api/server.go

```diff
@@ -203,9 +203,6 @@ func (s *Server) UploadVault(c echo.Context) error {
 	}
 
 	passwd := c.Request().Header.Get("x-password")
-	if passwd == "" {
```

Comment:
It's because we have unencrypted vault too.
If we don't remove this code, we can't use no-passwd vault.

Reply:
Yup, if the vault .bak is not password encrypted , then we should not allow it to upload to Vultisigner , because then anybody can use it , and download it